src/python/turicreate/test/test_style_transfer_tf.py

```diff
@@ -19,34 +19,34 @@
 from turicreate.toolkits._internal_utils import _raise_error_if_not_sframe, _mac_ver
 import coremltools
 
-from ..toolkits.style_transfer._tf_model_architecture import define_resnet as _define_resnet
-from ..toolkits.style_transfer._tf_model_architecture import define_instance_norm as _define_instance_norm
-
-from ..toolkits.style_transfer._model import InstanceNorm as _InstanceNorm
-from ..toolkits.style_transfer._model import ResidualBlock as _ResidualBlock
-from ..toolkits.style_transfer._model import Transformer as _Transformer
-
-import tensorflow as _tf
-import mxnet as _mx
-import numpy as _np
-
-
-# TODO: Refactor back into test_style_transfer
-
 _NUM_STYLES = 4
+_MARGIN_OF_ERROR = 5e-3
+
+def check_for_tensorflow():
+    try:
+        import tensorflow  
+        return False
+    except ImportError:
+        return True
 
+@unittest.skipIf(check_for_tensorflow(), 'Requires Tensorflow to be Installed')
 class StyleTransferTFTest(unittest.TestCase):
     @classmethod
     def setUpClass(self):
-        self.tf_style_image = np.random.random_sample((1, 256, 256, 3))
-        self.tf_content_image = np.ones((1, 256, 256, 3))
+        pass
 
-        self.mxnet_style_image = self.tf_style_image.transpose(0, 3, 1, 2)
-        self.mxnet_content_image = self.tf_content_image.transpose(0, 3, 1, 2)
+    def test_instance_norm_network(self):
+        import tensorflow as _tf
+        import mxnet as _mx
```

Comment:
What's the long-term plan for this test after we stop depending on MXNet?

Reply:
Not sure. Perhaps load saved weights, expected and expected outputs from variables (arrays) instead of using MxNet to generate them.